grails-app/controllers/au/org/ala/volunteer/ProjectController.groovy

```diff
@@ -472,7 +472,8 @@ class ProjectController {
             flash.message = "The expedition status was not able to be updated."
             render(view: '/notPermitted')
         } else {
-            if (!project.inactive) {
+            def isNotifyEnabled = (true == grailsApplication.config.notifications.project.enabled)
```

Comment:
It is possible `notifications.project.enabled` is a string boolean value if the configuration is set in properties file. This code might not work then. Similar comments below.